README.md

```diff
@@ -163,7 +173,7 @@ const main = ([p, s]: [parameter,storage]): ret =>
      });
 ```
 
-Explanations :
+Explanations:
 
 - the first line `#import "@ligo/fa/lib/fa2/nft/NFT.jsligo" "NFT"` imports the ligo FA library
 - `storage` definition is an extension of the imported library storage `(NFT.Ledger.t,NFT.Metadata.t,NFT.TokenMetadata.t,NFT.Operators.t,set<NFT.Storage.token_id>)`
```

Comment:
Need to be rephrased. Maybe to something like:
> - `storage` type is the storage imported from library: `(NFT.Ledger.t,NFT.Metadata.t,NFT.TokenMetadata.t,NFT.Operators.t,set<NFT.Storage.token_id>)` but with this and that because of this and that.
```

Reply:
Please also rework parameter. Because
> `parameter` has an entrypoint `AddAdministrator`to add new administrators
is kinda useless... Either your explain what is an administrator, and why we need them for the training, or you don't mention them in the explanations.